docs/design/coreclr/botr/corelib.md

```diff
@@ -54,7 +54,7 @@ If the only reason you're defining a FCall method is to call a native method, yo
 
```

Comment:
> There is a small variant of FCall called HCall (for Helper call) for implementing JIT helpers. The HCall is intended for doing things like accessing multi-dimensional array elements, range checks, etc. The only difference between HCall and FCall is that HCall methods won't show up in an exception stack trace.

The last sentence should be: "The only difference between HCall and FCall is that HCall methods are not mapped to a managed method in CoreLib."